### PR TITLE
use Joplin with oCIS

### PR DESCRIPTION
**Pull request by individual-it**
just a small blog post about how to "integrate" both products

### CODE REVIEW

File: src/assets/JoplinWithOcis/JoplinWithOcis.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Configure Joplin to work with ownCloud infinite scale (oCIS)
```
Comment (phil-davis) [inline src/assets/JoplinWithOcis/JoplinWithOcis.md:2]:
```suggestion
title: Configure Joplin to work with ownCloud Infinite Scale (oCIS)
```

Maybe the product name should have capitals?

File: src/assets/JoplinWithOcis/JoplinWithOcis.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+authorName: Artur Neumann
+authorAvatar: https://www.jankaritech.com/images/2019/06/11/p1070364-c-light-800.jpg
+authorLink: https://github.com/individual-it
+createdAt: Jun 7, 2023
+tags: joplin, owncloud, cloudstorage
+banner: https://github.com/JankariTech/blog/blob/fd8730dbc20a3855db82e8ae7a97f5c941e9e3fd/src/assets/JoplinWithOcis/joplin_with_ocis-header.png
+---
+
+I love to use [Joplin](joplinapp.org) to organize my notes. To synchronize the notes between different devices I have used so far the WebDAV sync option together with [ownCloud 10](github.com/owncloud/core/). Now [oCIS (ownCloud infinite scale)](https://owncloud.com/infinite-scale/) is the new cool kid in the cloud storage space and I would like to use it for syncing of Joplin.
```
Comment (phil-davis) [inline src/assets/JoplinWithOcis/JoplinWithOcis.md:11]:
```suggestion
I love to use [Joplin](joplinapp.org) to organize my notes. To synchronize the notes between different devices I have so far used the WebDAV sync option together with [ownCloud 10](github.com/owncloud/core/). Now [oCIS (ownCloud Infinite Scale)](https://owncloud.com/infinite-scale/) is the new cool kid in the cloud storage space and I would like to use it for syncing of Joplin.
```

File: src/assets/JoplinWithOcis/JoplinWithOcis.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Similar to ownCloud 10, oCIS offers a WebDAV API, but it has [disabled basic-auth by default and the docs discourage using it in production](https://doc.owncloud.com/ocis/next/deployment/services/s-list/auth-basic.html#introduction). Instead, oCIS implements the OIDC workflow for authentication. I tried to get the OIDC authentication into Joplin, but sadly my [PR](https://github.com/laurent22/joplin/pull/7400) for that was refused.
+
+The workaround is the "Share via link" function of oCIS. The generated links support basic-auth by default. (Don't fear your notes will be still very private).
```
Comment (phil-davis) [inline src/assets/JoplinWithOcis/JoplinWithOcis.md:15]:
```suggestion
The workaround is the "Share via link" function of oCIS. The generated links support basic-auth by default. (Don't fear your notes will still be very private).
```

File: src/assets/JoplinWithOcis/JoplinWithOcis.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+1. create a folder in oCIS (I've called it `joplin`)
+2. click on the "Add people" button of that folder, that will bring you to the sharing functions
+3. you should see the "Share via link" function
+4. create a new link
+5. change the permissions to "Anyone with the link can edit"
+6. additionally set a password ![password](/src/assets/JoplinWithOcis/password.png)
+7. to configure Joplin, open its synchronisation settings and:
+   1. select "WebDAV" as "Synchronisation target"
+   2. copy the whole link URL into "WebDAV URL"
+   3. take the last bit of the link URL (everything after tha last `/`) as "WebDAV username"
```
Comment (phil-davis) [inline src/assets/JoplinWithOcis/JoplinWithOcis.md:25]:
```suggestion
   3. take the last bit of the link URL (everything after the last `/`) as "WebDAV username"
```

File: src/assets/JoplinWithOcis/JoplinWithOcis.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      ![joplin settings](/src/assets/JoplinWithOcis/joplinSettings.png)
+   4. add your password into the "WebDAV password" field
+
+DONE!
+
+To access your notes, one would need to know the random link AND your self selected password - so it should be pretty safe.
+You could even have a different link, with a different password for every device you are using, so if you loose one device you would only have to delete that link to stop further syncing.
```
Comment (phil-davis) [inline src/assets/JoplinWithOcis/JoplinWithOcis.md:32]:
```suggestion
You could even have a different link, with a different password for every device you are using, so if you lose one device you would only have to delete that link to stop further syncing.
```

**Review by phil-davis — APPROVED** [review]

**Review by saw-jan — APPROVED** [review]
:+1: Great tool
Recently, I bumped into it and now using it to take standup texts. Syncs perfectly with mobile and desktop devices

**Pull request merged**